glean-core/rlb/src/lib.rs

```diff
@@ -45,7 +45,9 @@ use std::sync::Mutex;
 pub use configuration::Configuration;
 use configuration::DEFAULT_GLEAN_ENDPOINT;
 pub use core_metrics::ClientInfoMetrics;
-pub use glean_core::{global_glean, setup_glean, CommonMetricData, Error, Glean, Lifetime, Result};
+pub use glean_core::{
+    global_glean, setup_glean, CommonMetricData, Error, ErrorType, Glean, Lifetime, Result,
+};
```

Comment:
Should we also be exporting the metric traits so (for example) FOG's metrics need not `use` `glean_core` at all?

Reply:
I have no strong opinion on this, but does it change anything at all other than using `glean_core::traits` vs `glean::traits`? @badboy , thoughts?

Reply:
If we can drop a direct dependency on `glean_core` and instead only need to rely on `glean` we have less hassle of always updating the versions in sync.

Given that we just pulled a release we can do that in a followup though, no need to rush this out.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1679963